src/hipscat/io/file_io/file_io.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+
+import pandas as pd
+
+from hipscat.io.file_io.file_pointers import FilePointer
+
+
+def make_directory(file_pointer: FilePointer, exist_ok: bool = False):
+    """Make a directory at a given file pointer
+
+    Will raise an error if a directory already exists, unless `exist_ok` is True in which case
+    any existing directories will be left unmodified
+
+    Args:
+        file_pointer: location in file system to make directory
+        exist_ok: Default False. If false will raise error if directory exists. It true existing
+        directories will be ignored and not modified
+
+    Raises:
+        OSError
+    """
+    os.makedirs(file_pointer, exist_ok=exist_ok)
+
+
+def write_string_to_file(
+    file_pointer: FilePointer, string: str, encoding: str = "utf-8"
+):
+    """Write a string to a text file
+
+    Args:
+        file_pointer: file location to write file to
+        string: string to write to file
+        encoding: Default: 'utf-8', encoding method to write to file with
+    """
+    with open(file_pointer, "w", encoding=encoding) as file:
+        file.write(string + "\n")
```

Comment:
What's that newline about?

Reply:
I think I got this from the `write_json_file` method `write_metadata.py`. Probably best to remove it here and add it in the call there.